Tests/test_imagesequence.py

```diff
@@ -23,14 +23,11 @@ def test_sanity(self):
         self.assertEqual(index, 1)
 
     def _test_multipage_tiff(self, dbg=False):
-        # debug had side effect of calling fp.tell.
-        Image.DEBUG=dbg
         im = Image.open('Tests/images/multipage.tiff')
```

Comment:
I honestly can't see how `fp.tell` has any side effect.  Tests seem to be passing now so I'll leave it as it is.